README.md

```diff
@@ -296,8 +315,8 @@ logrusEntry.Debugf("hey!")
 
 Depending on how many features your logger is ready to provide it should implement one of:
 * `interface{ Printf(format string, args ...any)` or just be of type `func(string, ...any)`
-* [`Emitter`](https://github.com/facebookincubator/go-belt/blob/a80187cd561e4c30237aff5fccd46f06981d41e2/tool/logger/types/logger.go#L225)
-* [`CompactLogger`](https://github.com/facebookincubator/go-belt/blob/a80187cd561e4c30237aff5fccd46f06981d41e2/tool/logger/adapter/compact_logger.go#L17)
-* [`Logger`](https://github.com/facebookincubator/go-belt/blob/a80187cd561e4c30237aff5fccd46f06981d41e2/tool/logger/types/logger.go#L25)
+* [`Emitter`](https://github.com/facebookincubator/go-belt/blob/main/tool/logger/types/logger.go#L221-L251)
```

Comment:
you actually might not wanna use `main` here because line no's can change

Reply:
Just noticed the comment (was merging from the smartphone :( ). Yeah, I know about this problem. And that was the reason why I used commits before, but because of squashing commits it led to a broken link. I think I'll replace to commits back pretty soon.